src/elements/db/ElementQuery.php

```diff
@@ -2308,23 +2308,32 @@ private function _applyStructureParams(string $class): void
                 ['structureelements.structureId' => $this->structureId],
             ]);
         } else {
-            $this->query
-                ->addSelect(['structureelements.structureId'])
-                ->leftJoin(['structureelements' => Table::STRUCTUREELEMENTS], [
+            // $existsQuery = (new Query())
+            //     ->from([Table::STRUCTURES])
+            //     ->where('[[id]] = [[structureelements.structureId]]')
+            //     ->andWhere(['dateDeleted' => null]);
+            // $this->subQuery
+            //     ->addSelect(['structureelements.structureId'])
+            //     ->leftJoin(['structureelements' => Table::STRUCTUREELEMENTS], [
+            //         'and',
+            //         '[[structureelements.elementId]] = [[elements.id]]',
+            //         ['exists', $existsQuery],
+            //     ]);
+            $structureSubquery = (new Query())
+                ->addSelect([
+                    'structureelements.*',
```

Comment:
If possible I would explicitly define select fields here instead of using *. Makes it a little more clear what is being used.

Reply:
Yeah, that's a good point. I actually did specify the select fields in an earlier revision. I ended up opt against it because I wanted the subquery resultset to mimic as closely to the original left join statement as possible. 

The original was:
```
LEFT JOIN `structureelements` `structureelements` ON (`structureelements`.`elementId` = `elements`.`id`)
        AND EXISTS(SELECT * FROM `structures` WHERE (`id` = `structureelements`.`structureId`)  AND (`dateDeleted` IS NULL))```
 That would have returned all the columns from the `structureelements` table anyway.

Since there are so many much logic that happens afterwards that continues to modify the query object after this code, I decided it might be safer to keep the resultset the same in case there are some assumptions made later on in code. I did check the rest of the method, and it does seem to imply a pretty static set of fields.